test/asset_server_test.rb

```diff
@@ -198,6 +198,13 @@ def test_responds_not_found_if_path_contains_consecutive_periods
         end
       end
 
+      def test_responds_ok_if_path_contains_periods_that_not_follow_slash
+         ["/hello/path..with....periouds/file", "/hello/path/with..periods"].each do |dotty_path|
+          get dotty_path
+          assert_response_ok
```

Comment:
Consider using `assert_underlying_app_responded` instead of `assert_response_ok`